### ISSUE TITLE
AttributeError: 'module' object has no attribute 'NUM_CPUS'

### ISSUE THREAD

**Issue by w4-sjcho** [logentries/le#139]
I'm getting this error.

le version: 1.4.21
python version: 2.7.9
psutil version: 3.1.1

```
$ sudo le --debug --debug-events monitor
Traceback (most recent call last):
  File "/usr/bin/le", line 9, in <module>
    load_entry_point('logentries==1.4.21', 'console_scripts', 'le')()
  File "/usr/share/logentries/logentries/le.py", line 3140, in main
    return func(args[1:])
  File "/usr/share/logentries/logentries/le.py", line 2806, in cmd_monitor
    formatter, config.debug_metrics)
  File "/usr/share/logentries/logentries/metrics.py", line 459, in __init__
    self._items = self._instantiate(conf)
  File "/usr/share/logentries/logentries/metrics.py", line 482, in _instantiate
    items.append( CpuMetrics(conf.cpu == 'core', self._interval, self._transport, self._formatter))
  File "/usr/share/logentries/logentries/metrics.py", line 57, in __init__
    self._vcpus = _psutil_cpu_count()
  File "/usr/share/logentries/logentries/metrics.py", line 45, in _psutil_cpu_count
    return psutil.NUM_CPUS
AttributeError: 'module' object has no attribute 'NUM_CPUS'
Exception in thread Thread-1 (most likely raised during interpreter shutdown):
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 813, in __bootstrap_inner
  File "/usr/lib64/python2.7/threading.py", line 1085, in run
  File "/usr/share/logentries/logentries/le.py", line 1220, in send_stats
<type 'exceptions.AttributeError'>: 'NoneType' object has no attribute 'time'
```

**Comment by vilda** [comment]
Confirmed, I'll fix it in next PR

**Comment by vilda** [comment]
Fix is contained in PR #140. Thanks @w4-sjcho !
